queries.sql

```diff
@@ -0,0 +1,24 @@
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE
+      date_of_birth >= '2016-01-01'
+  and date_of_birth <= '2019-12-31';
+
+SELECT name FROM animals WHERE
+        neutered
+    and escape_attempts < 3;
```

Comment:
Please name all animals where `neutered = 'true'`. So the condition is that neutered has a boolean value `true`.

Reply:
the result of `neutered = 'true'` is the same as `neutered`